### DISCUSSION TITLE
laptop2

### DISCUSSION THREAD

**Discussion by ghost** [deaafrizal/forum#32] [All About Hardware ( PC, Laptop etc...)]
selamat siang kak @deaaprizal , mohon maaf mengganggu waktunya...  ingin bertanya mengenai spesifikasi dan meminta pendapat mengenai laptop yang 
awet dan cocok untuk dipakai ngoding.. sebelum nya saya sudah mempunyai beberapa pilihan yaitu:
•ASUS Vivobook 14 A1404VA-VIPS351 Ci3-1315U ram8GB ssd512GB
•LENOVO IdeaPad 3 14ITL6-82H701MXID i5-1155G7 ram8GB ssd512GB SSD Intel Iris Xe Graphics 
•Macbook air m1 ram8gb ssd256gb

klok untuk ngoding lebih bagus dan awet yang mana nggih kak?

**Comment by deaafrizal** [comment]
khusus ngoding tak di ragukan lagi bro yg ke 3.
